packages/knip/src/plugins/netlify/index.ts

```diff
@@ -0,0 +1,57 @@
+import { join } from '../../util/path.js';
+import { timerify } from '../../util/Performance.js';
+import { hasDependency, load } from '../../util/plugin.js';
+import { toProductionEntryPattern } from '../../util/protocols.js';
+import { extractFunctionsConfigProperty, validFunctionExtensions } from './helpers.js';
+import type { NetlifyConfig } from './types.js';
+import type { IsPluginEnabledCallback, GenericPluginCallback } from '../../types/plugins.js';
+
+// https://docs.netlify.com/ or https://docs.netlify.com/functions/get-started/?fn-language=ts
+
+const NAME = 'Netlify';
```

Comment:
Maybe it's more appropriate and future-proof to use name `Netlify Functions` and dir `netlify-functions`?

Reply:
I've actually changed this to not be specific to Netlify Functions as I read into the docs more, just Netlify in general.